tests/unit/common/monitored_resource_util/test_k8s_utils.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2019, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+try:
+    import mock
+except ImportError:
+    from unittest import mock
+
+import os
+import unittest
+
+from opencensus.common.monitored_resource import monitored_resource
+from opencensus.common.monitored_resource import k8s_utils
+
+
+class TestK8SUtils(unittest.TestCase):
```

Comment:
These were moved from `test_gcp_metadata_config.py`. Note that `instance_id` label doesn't appear anymore.